### PR TITLE
README: add GoDoc link

### PR DESCRIPTION
**Pull request by aalemayhu**
There is still a lot missing from the documentation, but what is there
might be useful for someone wanting to read the code from a browser
similar to lxr.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/cilium/cilium/pull/445%23discussion_r107964176%22%2C%20%22https%3A//github.com/cilium/cilium/pull/445%23discussion_r107965666%22%2C%20%22https%3A//github.com/cilium/cilium/pull/445%23discussion_r107965898%22%2C%20%22https%3A//github.com/cilium/cilium/pull/445%23issuecomment-289158753%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cilium/cilium/pull/445%23issuecomment-289158753%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Awesome%2C%20thanks%20%40scanf%20%22%2C%20%22created_at%22%3A%20%222017-03-24T22%3A32%3A37Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/1417913%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tgraf%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20055e656ec61fa8c26c210aa1adfa701bd7310fdc%20README.md%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cilium/cilium/pull/445%23discussion_r107964176%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20think%20it%20looks%20better%20right%20after%20the%20Go%20Report%20Card%22%2C%20%22created_at%22%3A%20%222017-03-24T17%3A57%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/5714066%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/aanm%22%7D%7D%2C%20%7B%22body%22%3A%20%22Amended.%22%2C%20%22created_at%22%3A%20%222017-03-24T18%3A03%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/925044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/scanf%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222017-03-24T18%3A04%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/925044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/scanf%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL6-13%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/cilium/cilium/pull/445#issuecomment-289158753'>General Comment</a></b>
- <a href='https://github.com/tgraf'></a> Awesome, thanks @scanf
- [x] <a href='#crh-comment-Pull 055e656ec61fa8c26c210aa1adfa701bd7310fdc README.md 1'></a> &nbsp;<b><a href='https://github.com/cilium/cilium/pull/445#discussion_r107964176'>File: README.md:L6-13</a></b>
- <a href='https://github.com/aanm'></a> I think it looks better right after the Go Report Card
- <a href='https://github.com/scanf'></a> Amended.

<a href='https://www.codereviewhub.com/cilium/cilium/pull/445?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cilium/cilium/pull/445?approve=1'></a>&nbsp;<a href='https://github.com/cilium/cilium/pull/445'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by aanm** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/cilium/cilium/pull/445'>Refresh</a>.<a href='#crh-update-none'></a>

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 [![Apache licensed](https://img.shields.io/badge/license-Apache-blue.svg)](https://github.com/cilium/cilium/blob/master/LICENSE)
```
Comment (aanm) [inline README.md:7]: I think it looks better right after the Go Report Card

**Review by aanm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aalemayhu) [reply README.md:7]: Amended.

Comment (aalemayhu) [inline README.md:7]: :+1:

**Review by aanm — APPROVED** [review]

**Comment by tgraf** [comment]
Awesome, thanks @scanf

**Pull request merged**